app/paid/paid.go

```diff
@@ -0,0 +1,122 @@
+package paid
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"io/ioutil"
+	"time"
+
+	"github.com/lbryio/lbrytv/internal/monitor"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+var logger = monitor.NewModuleLogger("paid")
+
+// Expfunc is a function type intended for CreateToken.
+// Should take stream size in bytes and return validity as Unix time
+type Expfunc func(uint64) int64
+
+// StreamToken contains stream hash and transaction id of a stream that has been purchased
+type StreamToken struct {
+	SDHash string `json:"sd"`
+	TxID   string `json:"txid"`
+	jwt.StandardClaims
+}
+
+type keyManager struct {
+	privKey         *rsa.PrivateKey
+	pubKeyMarshaled []byte
+}
+
+var km *keyManager
+
+// InitPrivateKey loads a private key from `file` for later token signing and derived pubkey distribution.
+// Should be generated as follows:
+// $ openssl genrsa -out privateKey.pem 2048
+func InitPrivateKey(file string) error {
```

Comment:
i recommend using io.Reader instead of a filename here. makes interface more flexible and testing easier